doc/source/rllib/rllib-cli.md

```diff
@@ -0,0 +1,336 @@
+---
```

Comment:
Awesomeness! Dumb question. Why not rst, but md?

Reply:
ha, thanks. well, a) I personally prefer markdown, b) the `margin` directive doesn't work in rst, and c) the yaml header in the file makes the whole thing an executable format. 

It's essentially a jupyter notebook in disguise. The serve team almost exclusively used `md` by now, and I just wanted to give a reference example for you guys to work off of.